src/reference/config/telemetry/exporters/exporter-syslog.md

```diff
@@ -0,0 +1,107 @@
+---
+shortTitle: syslog
+description: Zilla runtime syslog exporter
+category:
+  - Telemetry
+tag:
+  - Exporters
+---
+
+[Available in <ZillaPlus/>](https://www.aklivity.io/products/zilla-plus)
+{.zilla-plus-badge .hint-container .info}
+
+# Syslog Exporter
+
+Zilla runtime Syslog exporter
+
+```yaml {3}
+exporters:
+  syslog0:
+    type: syslog
+    options:
+      host: syslog-server
+      port: 514
+      protocol: tcp
+```
+
+## Configuration
+
+:::: note Properties
+
+- [options\*](#options)
+  - [options.host\*](#options-host)
+  - [options.port\*](#options-port)
+  - [options.protocol\*](#options-protocol)
+  - [options.trust](#options-trust)
+  - [options.trustcacerts](#options-trustcacerts)
+
+::: right
+\* required
+:::
+
+::::
+
+### options
+
+> `object`
+
+`syslog`-specific options.
+
+```yaml {4}
+options:
+  host: syslog-server
+  port: 514
+  protocol: tcp
+```
+
+```yaml {4}
+options:
+  host: syslog-server
+  port: 514
+  protocol: udp
+```
+
+```yaml {4}
+options:
+  host: syslog-server
+  port: 6514
+  protocol: tls
+  trust:
+    - syslog
+```
+
+#### options.host
+
+> `string`
+
+Host is the hostname of the syslog server.
+
+#### options.port
+
+> `integer`
+
+Port is the port of the syslog server.
+
+#### options.protocol
+
+> `string`
+
+Protocol is the protocol to use to communicate with the syslog server. Valid values are: `tcp`, `udp`, `tls`.
+
+#### options.trust
+
+> `array` of `strings`
+
```

Comment:
These are not really trust files. 

These are keys into the vault referenced on the binding, which could be a filesystem vault for a local pkcs12 keystore or an aws vault for remote pem format certificates stored in AWS secrets manager.

Perhaps we need to reference vault in the description and example here?